packages/components/src/components/chat-search/ContextStatus.vue

```diff
@@ -0,0 +1,209 @@
+<template>
+  <v-status-bar v-if="appmapStats && appmapStats.numAppMaps" level="info">
+    <template v-slot:header>
+      <div class="context__header">Navie is aware of your project</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__table mb-0">
+        <div class="context__table__header">Navie's context:</div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Runtime traces (AppMap recordings)</div>
+        </div>
+        <div class="context__table__row">
+          <v-success-icon class="success-icon" data-cy="success-icon" />
+          <div>Source code snippets</div>
+        </div>
+      </div>
+    </template>
+  </v-status-bar>
+
+  <v-status-bar v-else level="warning">
+    <template v-slot:header>
+      <div class="context__header">Navie is not aware of your project</div>
+    </template>
+    <template v-slot:body>
+      <div class="context__sub-header">
+        <div class="context__table">
+          <div class="context__table__header">Navie's context:</div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Runtime traces (AppMap recordings)</div>
+          </div>
+          <div class="context__table__row">
+            <v-fail-icon class="fail-icon" data-cy="fail-icon" />
+            <div>Source code snippets</div>
+          </div>
+        </div>
+        <p class="mt-0">
+          You can still ask Navie questions, but Navie does not have access to your project's data.
+        </p>
+        <template v-if="!appmapYmlPresent">
+          <p class="mb-0 mt-0" data-cy="status-no-config">
+            You need to
+            <a href="#" data-cy="install-guide" @click.prevent="openInstallInstructions">
+              finish setting up AppMap
+            </a>
+            and then
+            <a href="#" @click.prevent="openRecordInstructions" data-cy="record-guide">
+              create AppMap recordings
+            </a>
+            for Navie to know about your project.
+          </p>
+        </template>
+        <p
+          v-else-if="appmapStats && appmapStats.numAppMaps === 0"
+          class="mt-0 mb-0"
+          data-cy="status-no-data"
+        >
+          You need to
+          <a href="#" @click.prevent="openRecordInstructions" data-cy="record-guide">
+            record runtime data
+          </a>
+          for Navie to know about your project.
+        </p>
+      </div>
+    </template>
+  </v-status-bar>
+</template>
+
+<script lang="ts">
+// @ts-nocheck
```

Comment:
I've been trying not to do this because it completely hides compiler errors.

Reply:
Cool, I deleted this line :+1: